mountpoint-s3-fs/src/prefix.rs

```diff
@@ -26,6 +26,10 @@ impl Prefix {
         }
     }
 
+    pub fn empty() -> Self {
+        Self::new("").unwrap()
```

Comment:
Maybe just
```suggestion
        Self {
            path: String::new(),
        }
```
to avoid `unwrap`